add another example to clarify the difference between "validity" and "max-allowed-end" (#54)
netmod-wg/schedule-yang: draft-ietf-netmod-schedule-yang.md

  }
}
~~~~
{: #ex-0 title="Generic Parameters with "max-allowed-end" for Schedule Validation"}

>     To illustrate the subtle difference between "max-allowed-end" and "validity" parameters,

Reviewer: ```suggestion
    To illustrate the difference between "max-allowed-end" and "validity" parameters,
```

    {{ex-00}} illustrates the example of a requested schedule that needs to start no earlier than

>     08:00 AM, January 1, 2025 (Beijing time), schedule requests that fail to meet the

Reviewer: ```suggestion
    08:00 AM, January 1, 2025 (Beijing time). Schedule requests that fail to meet the
```

    requirements are discarded with warning messages. The requested schedule may end

>     after 8:00 PM, January 31, 2025, but any occurrences generating

Reviewer: ```suggestion
    after 8:00 PM, January 31, 2025, but any occurrences that are generated
```

>     after that time would be considered invalid.

Reviewer: ```suggestion
    after that time would be considered as invalid.
```
